app/assets/stylesheets/guitars.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the guitars controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
IMHO you don't need this file, we will use styles from the JS app :)

Reply:
OK 👍🏼 I'll remove it